include/z64item.h

```diff
@@ -195,18 +195,18 @@ typedef enum {
     /* 0x2F */ ITEM_COJIRO,
     /* 0x30 */ ITEM_ODD_MUSHROOM,
     /* 0x31 */ ITEM_ODD_POTION,
-    /* 0x32 */ ITEM_SAW,
-    /* 0x33 */ ITEM_SWORD_BROKEN,
+    /* 0x32 */ ITEM_POACHERS_SAW,
+    /* 0x33 */ ITEM_BROKEN_GORONS_SWORD,
     /* 0x34 */ ITEM_PRESCRIPTION,
-    /* 0x35 */ ITEM_FROG,
-    /* 0x36 */ ITEM_EYEDROPS,
+    /* 0x35 */ ITEM_EYEBALL_FROG,
+    /* 0x36 */ ITEM_EYE_DROPS,
     /* 0x37 */ ITEM_CLAIM_CHECK,
-    /* 0x38 */ ITEM_BOW_ARROW_FIRE,
-    /* 0x39 */ ITEM_BOW_ARROW_ICE,
-    /* 0x3A */ ITEM_BOW_ARROW_LIGHT,
+    /* 0x38 */ ITEM_BOW_FIRE,
+    /* 0x39 */ ITEM_BOW_ICE,
+    /* 0x3A */ ITEM_BOW_LIGHT,
```

Comment:
This change is kind of weird since it's not a fire bow item, it's e.g. a Bow + Fire Arrows, hence BOW_ARROW_FIRE

Reply:
I feel like `ITEM_BOW_ARROW_FIRE` makes it too similar to `ITEM_ARROW_FIRE` and it could be confused for the fire arrows item

Reply:
Bump: Is this considered resolved?
I'll assume yes in like 7 days or so